pkg/kds/types.go

```diff
@@ -0,0 +1,30 @@
+// Package kds provides support of Kuma Discovery Service, extension of xDS
+package kds
+
+import (
+	"github.com/Kong/kuma/pkg/core/resources/apis/mesh"
+	"github.com/Kong/kuma/pkg/core/resources/model"
+)
+
+const (
+	prefix = "type.googleapis.com/kuma.mesh.v1alpha1."
+)
+
+var (
+	SupportedTypes = []model.ResourceType{
+		mesh.MeshType,
+		mesh.DataplaneType, // for Ingress
```

Comment:
OK, so this is for the Global to push ingresses of the "other" locals?

Reply:
not only `// for Ingress` I expect this code to be used for the server/client in the other direction

Reply:
Yes, that's right. I delete the comment